scripts/UnifiedLogReader.py

```diff
@@ -292,9 +292,9 @@ def WriteLogEntry(self, log):
                     msg += log[22]
 
                     self._file_object.write((
-                        '{time:26} {li[4]:<#10x} {li[5]:11} {li[6]:<#20x} '
-                        '{li[8]:<6} {li[10]:<4} {message}').format(
-                            li=log, time=log[3], message=msg))
+                        '{time} {li[4]:<#10x} {li[5]:11} {li[6]:<#20x} '
+                        '{li[8]:<6} {li[10]:<4} {message}\n').format(
+                            li=log, time=log[3], message=msg.replace('\n','')))

```

Comment:
Please change `message=msg.replace('\n','')` 
to `message=msg.replace('\n',',')`
This way data from two lines will still be somewhat separated.

Reply:
You can just make another commit and it will automatically get added to the pull request.

Reply:
Thanks. Done.